apps/portal/js/wizard.js

```diff
@@ -93,7 +94,8 @@ $("#previewChart").click(function() {
             } else {
                 //TODO DOn't do this! read this from a config file
                 subscribe(streamId.split(":")[0], streamId.split(":")[1], '10', 'carbon.super',
```

Comment:
@suhothayan 
I checked the code and dashboard, this value is used to created the directory for domain where gadgets are stored. This does not work in tenant mode. I created a JIRA(CEP-1291) for above bug.